### ISSUE TITLE
Inconsistent flag to specify registries config

### ISSUE THREAD

**Issue by janpieper** [argoproj-labs/argocd-image-updater#290]
**Describe the bug**

The flag to specify the registries config file is inconsistent.

**To Reproduce**

```
$ argocd-image-updater run --help | grep registries-conf
      --registries-conf-path string      path to registries configuration file (default "/app/config/registries.conf")
```

vs.

```
$ argocd-image-updater test --help | grep registries-conf
      --registries-conf string      path to registries configuration
```

**Expected behavior**

Have `--registries-conf` **OR** `--registries-conf-path` for both commands.

**Additional context**

_None_

**Version**

0.11.0

**Logs**

_None_

**Comment by jannfis** [comment]
Good catch. I think for better backwards compatibility, we should reuse what's defined for the `run` command, so `--registries-conf-path`.

**Comment by janpieper-gcx** [comment]
Fixed by #295

**Comment by janpieper-gcx** [comment]
@jannfis Hmmm, I cannot close this issue 🤔
